src/GSMConnectionManager.cpp

```diff
@@ -106,11 +105,23 @@ void GSMConnectionManager::check() {
           debugMessage(DebugLevel::Verbose, "Connected to Cellular Network");
         }
         break;
+      case NetworkConnectionState::GETTIME: {
```

Comment:
I believe these were taken out by @ilcato to optimise.
may I ask why they have been reintroduced?
I like them there for future use

Reply:
Same reason for reintroducing them as in the statement above - additionally I'd like to quote Donald Knuth who said **"Premature optimization is the root of all evil"** :smile: We are not in High-Frequency-Trading, clarity and expressiveness of code should matter most.